bin/pycbc_live

```diff
@@ -548,7 +563,13 @@ class LiveEventManager(object):
                     if hasattr(event, 'p_terr') and event.p_terr is not None:
                         single['p_terr'] = event.p_terr
                 live_ifos = [ifo for ifo in sld if 'snr_series' in sld[ifo]]
-                gdb_upload_thread = threading.Thread(target = self.upload_in_thread, args = (event, fname, comment, single, live_ifos, [ifo], upload_checks, optimize_snr_checks, args.gracedb_server, args.gracedb_search,))
+                thread_args = (
+                    event, fname, comment, coinc_results, live_ifos, [ifo],
```

Comment:
Careful, I think you are changing the arguments here!

Reply:
ah, I missed the coinc -> single ...